rom/dev/doc/error-attribution.md

```diff
@@ -0,0 +1,232 @@
+# **Fatal Errors**
```

Comment:
Is there any information here that's not already in `error/src/lib.rs`? Manually maintaining a duplicate seems like a waste of time to me...

Reply:
Errors in lib.rs don't have an attribution of fatal vs. non-fatal. Eventually, this document can get published as official.

Reply:
Discussed offline; we'll keep this for now and auto-generate it later. At that time, this file should be removed.